meson_options.txt

```diff
@@ -1,2 +1,3 @@
 option('gtkver', type: 'integer', value: 3, description: 'GTK+ version')
 option('docs', type: 'boolean', value: false, description: 'add html documentation and examples')
+option('bash', type: 'boolean', value: false, description: 'run commands using bash -c')
```

Comment:
Should default be true? I suppose not as old behaviour would still work.

Reply:
I want to avoid risk in "traditional" Puppy builds, in case g_spawn_sync() doesn't behave 100% the same way as system() (signal handlers, signal mask, etc').

Reply:
Makes sense